storage/innobase/trx/trx0sys.cc

```diff
@@ -419,6 +419,8 @@ void trx_sys_update_wsrep_checkpoint(
 #endif /* !UNIV_DEBUG */
 #endif
   {
+    if(wsrep_implicit_transaction(current_thd)) return ;
```

Comment:
Please give a space after `if`
```suggestion
    if (wsrep_implicit_transaction(current_thd)) return ;
```